workspace/__lib__/xod-dev/text-lcd/print-at(text-lcd-i2c-device)/patch.cpp

```diff
@@ -1,40 +1,37 @@
 
-struct State {
-};
-
-{{ GENERATED_CODE }}
-
-void printAt(LiquidCrystal_I2C* lcd, uint8_t rowIndex, uint8_t posIndex, uint8_t len, XString str) {
-    lcd->setCursor(posIndex, rowIndex);
-    uint8_t whitespace = len;
-    for (auto it = str.iterate(); it && whitespace > 0; ++it, --whitespace)
-        lcd->write(*it);
+node {
+    void printAt(LiquidCrystal_I2C* lcd, uint8_t rowIndex, uint8_t posIndex, uint8_t len, XString str) {
```

Comment:
```suggestion
    static void printAt(LiquidCrystal_I2C* lcd, uint8_t rowIndex, uint8_t posIndex, uint8_t len, XString str) {
```

Because no dependency on `this` here. Saves a few call instructions.